src/common/AbstractObject.cpp

```diff
@@ -43,6 +43,7 @@ under the License.
 using namespace std;
 using namespace COMMON_NS;
 using namespace gsoap_resqml2_0_1;
+using namespace gsoap_eml2_1;
```

Comment:
Probably a bit dangerous to include both.
Down the road, i think I will include non of them and for to write each time the right namespace.
Not a blocker for this PR. It is just to let you know to preferably write the right namespace in this file when possible with long term in mind.

Reply:
Ok. Done.